### DISCUSSION TITLE
How to open the editor after I compiled?

### DISCUSSION THREAD

**Discussion by hongchangchun** [o3de/o3de#2224] [Help Wanted]
I follow the readme and run the edit.exe or o3de.exe in the bin/profile/ folder, create a new project, and try to open it , but failed.

**Comment by hongchangchun** [comment]
any one can open the editor ?  I'm confused about that I cannot open the editor without a project?

**Reply by amznestebanpapp** [reply]
You need to have a project configured to open the Editor.

The Editor/AssetProcessor and other tools configure the plugins that they load based on the project that is selected. If no project is selected, there are no plugins to load and nothing to show. 

o3de can open without a project. The expected workflow is that if there is no project selected, o3de opens and lets you pick a project or create a new one.

**Comment by Pshaowang** [comment]
Hi @hongchangchun, please let us know if you still have issues with opening the editor without a selected project. 

Thanks.

**Comment by bearbones** [comment]
I have the same problem, workflow as follows:

1. Compile with cmake, then navigate to o3de\out\build\o3de-x64-Release\bin\release and double click o3de.exe
2. Window with two tabs, Project and Engine. Change default paths in Engine (because C:\ doesn't have enough space).
3. Create symbolic link from C:\Users\<me>\\.o3de to the drive with enough space
4. Click New Project button, create new project, see a dialog that says I need to "rebuild" it, click the Build button on the new project's icon, wait for build to finish.
5. Click Open Editor on the new project's icon, see launcher splash for the editor, says it's Connecting with the Asset Processor and then Loading Plugins, at which point it vanishes and nothing else appears to happen.


I don't yet know where to find the logs that OP posted, but I'll add them once I find out. I do have the project build log:
[CMakeProjectBuildError.log](https://github.com/o3de/o3de/files/6871727/CMakeProjectBuildError.log)

This is a screenshot of the [Open Editor button](https://imgur.com/a/SOAeVHY).

I then tried the command line instructions from the "Setting up a new project" section under a new project folder (bbbbbb). The same issue occurs when clicking the Open Editor button after adding it as an existing project. I'll note that I added it as an existing project before the build command finished. Since the output of the command had said it built bbbbbb.GameLauncher.exe, I navigated there and ran that, which opened a blank window (O3DE Launcher) and an error dialog: [screenshot](https://imgur.com/TaTiaWY).

**Reply by bearbones** [reply]
Update: tried running AssetProcessor.exe from the project build folder. Closing that and deleting the other process before starting it from the project folder again seems to have fixed that part, maybe that was why the launch was stunted

![image](https://user-images.githubusercontent.com/57124166/126876262-dfbdcac5-3c65-4ea8-8189-dd14cfdd2df4.png)

**Reply by bearbones** [reply]
Okay, I think running AssetProcessor.exe (as the only process of its kind) from the project build\bin folder is the fix. After it finished processing all the assets, I ran Editor.exe from the project build\bin folder and got the Editor!
![image](https://user-images.githubusercontent.com/57124166/126876594-acc1529c-da55-4bed-a8d1-9be2d9a2bad4.png)

**Reply by Pshaowang** [reply]
That's great to hear! Glad you are able to run the editor now.
